src/test/java/com/box/sdk/BoxFileTest.java

```diff
@@ -644,6 +644,30 @@ public void fileLockAndUnlockSucceeds() {
         BoxFile.Info updatedFileInfo = uploadedFile.getInfo("lock");
         assertThat(updatedFileInfo.getLock(), is(nullValue()));
 
+        uploadedFile.lock(false);
```

Comment:
Hey @meavenson, thank you so much for the pull request! 

This looks great, the only thing is it would be great to assert that getIsDownloadPrevented() asserts to true here as it is false by default and this way we can ensure that the out of the ordinary case passes. Otherwise everything else looks great and we can merge this in today.  

Thank you!